data/unaligned_labeled_mask_cls_dataset.py

```diff
@@ -0,0 +1,45 @@
+from data.unaligned_labeled_mask_dataset import UnalignedLabeledMaskDataset
+
+
+class UnalignedLabeledMaskClsDataset(UnalignedLabeledMaskDataset):
```

Comment:
Maybe we could "gather all the datasets into one" ? :) to discuss

Reply:
yes of course but I don't know i'll do that in this PR or in another one actually

Reply:
Better to do things incrementally I think

Reply:
I agree